### PR TITLE
Add blktap-3.30.0-allocate-sufficient-space-in-normalize_path.backport.patch

### PR DESCRIPTION
**Pull request by Wescoeur**
…patch to avoid invalid write

### CODE REVIEW

**Comment by Wescoeur** [comment]
Build: http://koji.xcp-ng.org/taskinfo?taskID=18902

File: SPECS/blktap.spec [hunk 1]
```diff
@@ -13,8 +13,9 @@ Provides: gitsha(https://code.citrite.net/rest/archive/latest/projects/XS/repos/
 
 # XCP-ng patches
 
-# The DRBD patch must be removed after the upstream release of the version 3.42.
+# The DRBD patches below must be removed after the upstream release of the version 3.43.
```
Comment (stormi) [inline SPECS/blktap.spec:16]: The comment say "DRBD patches" but the second patch does not seem to be related to DRBD

File: SPECS/blktap.spec [hunk 2]
```diff
@@ -13,8 +13,9 @@ Provides: gitsha(https://code.citrite.net/rest/archive/latest/projects/XS/repos/
...
 Patch1000: blktap-3.30.0-drbd-support.XCP-ng.patch
+Patch1001: blktap-3.30.0-allocate-sufficient-space-in-normalize_path.XCP-ng.patch
```
Comment (stormi) [inline SPECS/blktap.spec:18]: The patch seems to be a backport, so according to our patch naming rules it should end in `.backport.patch` rather than `.XCP-ng.patch`.

**Review by stormi — COMMENTED** [review]
If possible, it would be good to add a few words about the invalid write the patch fixes in the commit message (when it happens and why), since we don't have access to the CA ticket the commit references.

File: SPECS/blktap.spec (hunk 1 above)
Comment (Wescoeur) [reply SPECS/blktap.spec:16]: The second patch is directly linked to the first.

**Comment by Wescoeur** [comment]
I think the content of the patch is clear, there is not enough space in the chain for the last "/". :)

**Comment by stormi** [comment]
Ok, but the patch contents does not display in git history so there will be nothing in it that explains the WHY of the commit exactly.

**Comment by stormi** [comment]
The DCO check has failed.

**Comment by Wescoeur** [comment]
Fixed, it's the first time I use it. :)

**Comment by stormi** [comment]
I've merged it to master too.

**Comment by stormi** [comment]
Built in koji, target v8.2-testing: https://koji.xcp-ng.org/buildinfo?buildID=2050
